locales/de.json

```diff
@@ -38,5 +38,11 @@
     "Next post": "Nächster Beitrag",
     "Previous post": "Vorheriger Beitrag",
     "Sign in": "Anmelden",
-    "Sign up": "Mitglied werden"
+    "Sign up": "Mitglied werden",
```

Comment:
`Sign up` I'd rather translate as `Registrieren`, since it doesn't say `Become a member`, or? What about using `Sign in : Einloggen` and `Sign up : Anmelden`?

Reply:
Yeah you're right. I personally like "Mitglied werden" more since it is easier to comprehend than using `Einloggen` and ` Anmelden` – i always have to stop and think about which one is which. Since Anmelden can mean Register and Login at the same time, i find it too confusing.

I'd go with `Sign Up: Registrieren` and `Sign in: Einloggen`, what do you think?